type-definitions/most.js.flow

```diff
@@ -0,0 +1,410 @@
+    fn: (a: A, b: B) => R,
+    streams: [Stream<B>]
+  ): Stream<R>;
+  combineArray<B, C, R>(
+    fn: (a: A, b: B, c: C) => R,
+    streams: [Stream<B>, Stream<C>]
+  ): Stream<R>;
+  combineArray<B, C, D, R>(
+    fn: (a: A, b: B, c: C, d: D) => R,
+    streams: [Stream<B>, Stream<C>, Stream<D>]
+  ): Stream<R>;
+  combineArray<B, C, D, E, R>(
+    fn: (a: A, b: B, c: C, d: D, e: E) => R,
+    streams: [Stream<B>, Stream<C>, Stream<D>, Stream<E>]
+  ): Stream<R>;
+  combineArray<V, R>(
+    fn: (a: A, ...rest: V[]) => R,
+    streams: Stream<V>[]
+  ): Stream<R>;
+
+  scan<B>(f: (b: B, a: A) => B, b: B): Stream<B>;
+  loop<S, B>(f: (seed: S, a: A) => SeedValue<S, B>, seed: S): Stream<B>;
+
+  concat(s2: Stream<A>): Stream<A>;
+  startWith(a: A): Stream<A>;
+
+  filter(p: (a: A) => boolean): Stream<A>;
+  skipRepeats(): Stream<A>;
+  skipRepeatsWith(eq: (a1: A, a2: A) => boolean): Stream<A>;
+
+  take(n: number): Stream<A>;
+  skip(n: number): Stream<A>;
+  takeWhile(p: (a: A) => boolean): Stream<A>;
+  skipWhile(p: (a: A) => boolean): Stream<A>;
+  skipAfter(p: (a: A) => boolean): Stream<A>;
+  slice(start: number, end: number): Stream<A>;
+
+  until(signal: Stream<any>): Stream<A>;
+  takeUntil(signal: Stream<any>): Stream<A>;
+  since(signal: Stream<any>): Stream<A>;
+  skipUntil(signal: Stream<any>): Stream<A>;
+  during(timeWindow: Stream<Stream<any>>): Stream<A>;
+  throttle(period: number): Stream<A>;
+  debounce(period: number): Stream<A>;
+
+  timestamp(): Stream<TimeValue<A>>;
+  delay(dt: number): Stream<A>;
+
+  await<B>(this: Stream<Promise<B>>): Stream<B>;
+  awaitPromises<B>(this: Stream<Promise<B>>): Stream<B>;
+
+  sample<B, C, R>(
+    fn: (b: B, c: C) => R,
+    b: Stream<B>,
+    c: Stream<C>
+  ): Stream<R>;
+  sample<B, C, D, R>(
+    fn: (b: B, c: C, d: D) => R,
+    b: Stream<B>,
+    c: Stream<C>,
+    d: Stream<D>
+  ): Stream<R>;
+  sample<B, C, D, E, R>(
+    fn: (b: B, c: C, d: D, e: E) => R,
+    b: Stream<B>,
+    c: Stream<C>,
+    d: Stream<D>,
+    e: Stream<E>
+  ): Stream<R>;
+
+  sampleWith(sampler: Stream<any>): Stream<A>;
+
+  zip<B, R>(
+    fn: (a: A, b: B) => R,
+    b: Stream<B>
+  ): Stream<R>;
+  zip<B, C, R>(
+    fn: (a: A, b: B, c: C) => R,
+    b: Stream<B>,
+    c: Stream<C>
+  ): Stream<R>;
+  zip<B, C, D, R>(
+    fn: (a: A, b: B, c: C, d: D) => R,
+    b: Stream<B>,
+    c: Stream<C>,
+    d: Stream<D>
+  ): Stream<R>;
+  zip<B, C, D, E, R>(
+    fn: (a: A, b: B, c: C, d: D, e: E) => R,
+    b: Stream<B>,
+    c: Stream<C>,
+    d: Stream<D>,
+    e: Stream<E>
+  ): Stream<R>;
+
+  recoverWith<B>(p: (a: B) => Stream<A>): Stream<A>;
+  multicast(): Stream<A>;
+
+  thru<B>(transform: (stream: Stream<A>) => B): B;
+}
+
+declare export function just<A>(a: A): Stream<A>;
+declare export function of<A>(a: A): Stream<A>;
+declare export function empty(): Stream<any>;
+declare export function never(): Stream<any>;
+declare export function from<A>(as: A[] | Iterable<A> | Observable<A>): Stream<A>;
```

Comment:
At first, I thought `A[]` isn't valid flow array syntax, but [turns out it is](https://flow.org/en/docs/types/arrays/#toc-array-type-shorthand-syntax)!  TIL 😄 :+1: